### ISSUE TITLE
Possible rewrite to function as a proper library?

### ISSUE THREAD

**Issue by ghost** [CarvellScott/python_intensifies#3]
I was trying to use it as a normal library within a program by using ``import`` but I'm kinda lost since docs don't really exist

**Comment by CarvellScott** [comment]
Hey, sorry that it took me so long to see this. Honestly, using it as a library isn't something I initially had in mind, so most documentation is kind of just in the form of arguments. Been thinking about refactoring/rewriting this lately anyway. Is there a preferred way you'd like to call it?

**Comment by ghost** [comment]
I don't think it should be a brand new project, it's possible to keep the command line functionality and add a proper library-like API.

I'll start working on a POC

**Comment by CarvellScott** [comment]
I had kinda already started 2 years ago. I polished it up a bit and pushed a few moments ago if you'd like to take a look:
https://github.com/CarvellScott/python_intensifies/tree/api_refactor
